### DISCUSSION TITLE
VSCode plugin: create file with markdown link when pasting an image

### DISCUSSION THREAD

**Discussion by liquidcarbon** [quarto-dev/quarto-cli#8962] [Feature Requests]
### Description

In Markdown language mode, this happens automagically, and it's super convenient.

Is there a way to make it happen in Quarto mode?

![Recording 2024-03-02 at 16 43 13](https://github.com/quarto-dev/quarto-cli/assets/47034358/828822d9-0efe-410a-9dd5-133867ff2ea9)

**Comment by mcanouil** [comment]
Please search and open an issue (if it does not exist) on <https://github.com/quarto-dev/quarto> for the VSCode extension.

**Comment by liquidcarbon** [comment]
thank you!
indeed, https://github.com/quarto-dev/quarto/issues/326 is a near duplicate.
